### PR TITLE
Add German translation (mostly from original German version)

### PR DESCRIPTION
**Pull request by BaldwinTechnology**
This is the german version of winfile

### CODE REVIEW

**Comment by BaldwinTechnology** [comment]
I see I forgot to set the width of the controls.

**Comment by craigwi** [comment]
The about menu item was removed from the context menu (what is named "Dummy Popup"); can you remove that from your version?  

Otherwise, look fine to me.

**Comment by craigwi** [comment]
Oh and please do not use tabs; 4 spaces == tab (e.g., wfloc.c).

**Comment by BaldwinTechnology** [comment]
I removed the entry and will add a new pull request as soon as I have corrected the Dialog width.

@craigwi: Where did I used tabs? I checked my files and there are only spaces.

File: src/wfloc.c [hunk 1]
```diff
@@ -20,6 +20,7 @@ LPCTSTR szLCIDs[] = {
     TEXT("he-IL"),    // Hebrew, Israel
     TEXT("ja-JP"),    // Japanese, Japan
     TEXT("pl-PL"),    // Polish, Poland
+	TEXT("de-DE"),    // German, Germany
```
Comment (mattn) [inline src/wfloc.c]: please fix indentatino

Comment (BaldwinTechnology) [reply src/wfloc.c]: Oh, I see.

**Comment by craigwi** [comment]
Fixed in #176.

**Pull request closed without merging**